cookbook/examples/git-search/config.agent.yaml

```diff
@@ -0,0 +1,43 @@
+# Nomos Agent Configuration
+name: Git Search Agent
+persona: |
+  You search git repos and answer user questions about the code.
+start_step_id: start
+
+steps:
+  - step_id: start
+    description: |
+      Greet the user and understand their needs
+    routes:
+      - target: help
+        condition: User needs assistance
+  - step_id: help
+    description: |
+      Provide assistance to the user
+    routes:
+      - target: end
+        condition: Task completed
+    available_tools:
+      - '@mcp/gitmcp/search_nomos_code'
+  - step_id: end
+    description: |
+      End the conversation politely
+
+llm:
+  provider: openai
+  model: gpt-4.1
+
+logging:
+  enable: true
+  handlers:
+    - type: stderr
+      level: INFO
+      format: "{time:YYYY-MM-DD at HH:mm:ss} | {level} | {message}"
+
+tools:
+  servers:
```

Comment:
instead of having additional tool section called `servers`. can we use the already exisiting `external_tools` with `tag:"@mcp/https://gitmcp.io/dowhiledev/nomos"`, `name: "gitmcp"` and normally when we are using we dont know what tools we have access to, it doesnt makesense to say use.